modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/data/handler/BatchEnginePortletDataHandlerUtil.java

```diff
@@ -100,6 +100,22 @@ public static Map<String, Serializable> buildExportParameters(
 			"itemClassName", exportImportDescriptor.getItemClassName()
 		).put(
 			"itemModelName", exportImportDescriptor.getItemModelName()
+		).put(
+			"siteExternalReferenceCode",
+			() -> {
+				Map<String, String[]> map =
```

Comment:
Is it possible to centralize the calculation of the siteExternalReferenceCode here? Just asking, I saw you added also in the `exportDeletionSystemEvents`, so not sure if what I say is possible